packages/protocol/socket.ts

```diff
@@ -8,7 +8,7 @@ import {
   CommandParams,
   CommandResult,
 } from "@recordreplay/protocol";
-import { isMock, mockEnvironment, waitForMockEnvironment } from "ui/utils/environment";
+import { isMock, mockEnvironment, waitForMockEnvironment } from "shared/utils/environment";
```

Comment:
I bet you can avoid importing `shared/utils/environment` from `packages/protocol`. Might need other utils, but probably not this one

Reply:
The "packages/shared" package is based on the idea from React's package of a similar name– things that are used by multiple projects in the monorepo. Any code in the package would get inlined during build (aka there's no "shared" NPM package). I think conceptually, this may make sense for Replay too.

Now maybe environment isn't a good use case for that. Totally.

I don't want to get too hung up on designing a perfect end state for this package though, because I don't think it's compelling to publish and consume anywhere other than Replay _for now_. Just taking baby steps. :)

Reply:
Removed in 41378d5a822ce33bb5e23cec70df202a1ab2ef58